src/cli/backup/groups_e2e_test.go

```diff
@@ -145,20 +145,17 @@ func runGroupsBackupCategoryTest(suite *BackupGroupsE2ESuite, category path.Cate
 
 	result := recorder.String()
 	t.Log("backup results", result)
-
-	// as an offhand check: the result should contain the m365 group id
-	assert.Contains(t, result, suite.its.group.ID)
 }
 
 func (suite *BackupGroupsE2ESuite) TestGroupsBackupCmd_groupNotFound_channelMessages() {
-	runGroupsBackupGroupNotFoundTest(suite, channelMessages)
```

Comment:
path type is - `channelMessages` where as the input to CLI command requires- `messages`